src/expressServer.js

```diff
@@ -82,6 +79,7 @@ app.use(
     stream: { write: (message) => logger.info(message.trim()) },
   })
 );
+app.use('/recommendations', recommendationRoutes);
```

Comment:
```suggestion
app.use('api/v1/recommendations', recommendationRoutes);
```

And let's please move it to line 155, so that this line occurs _after_ all our middleware that we're using to keep our API endpoints secure, logged, and handled in a consistent manner.

Reply:
Corrected and moved on line 154, commit 080bc44e62efd44e56c1b970ed3fb373362d1c38